src/chrome/MiniButton.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import classNames from 'classnames'
+
+import Icon, { IconSize } from './Icon'
+
+import { ButtonType, generateButtonTypeClasses } from './Button'
+
+export interface MiniButtonProps {
+  // icon is required
+  icon: string
```

Comment:
this type should really be an enum of all possible icon strings, but that isn't defined in the `Icon` component yet. I'll file an issue